### PR TITLE
All products page

### PR DESCRIPTION
**Pull request by Yuliia143**

### CODE REVIEW

File: src/app/breadcrumbs/breadcrumbs.component.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
 import { Component, OnInit } from '@angular/core';
+import { Router, RoutesRecognized } from '@angular/router';
+
+interface BreadCrumbInterface {
```
Comment (maksym-syniuk) [inline src/app/breadcrumbs/breadcrumbs.component.ts]: That would be better to have a folder in your project, called "Interfaces" or "Models" and inside this folder create files with corresponding names, in your case: "breadcrumb.interface.ts" (if it was model, that would be "breadcrumb.model.ts"), there you just need to export this interface and import in every file you need.

File: src/app/breadcrumbs/breadcrumbs.component.ts [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
-  }
+    constructor(private route: Router) {
+    }
 
+    ngOnInit(): void {
+        this.route.events.subscribe((data) => {
+            if (data instanceof RoutesRecognized) {
+                this.breadcrumbs = data.state.root.firstChild.data.breadcrumb;
+            }
+        });
```
Comment (maksym-syniuk) [inline src/app/breadcrumbs/breadcrumbs.component.ts:26]: Always try to split logic to different methods, especially if it is `ngOnInit`. Also, call this method semantically, for example in this case :
 ``` ts
ngOnInit(): void {
  this.subscribeToRouteChanges();
}

public subscribeToRouteChanges(): void {
  this.route.events.subscribe((data) => {
    if (data instanceof RoutesRecognized) {
      this.breadcrumbs = data.state.root.firstChild.data.breadcrumb;
    }
  });
}

File: src/app/header/header.component.scss [hunk 3]
```diff
@@ -1,18 +1,136 @@
...
-      justify-content: space-around;
+      align-items: center;
+      font-family: $base-font-secondary;
+      font-weight: $font-weight-semi-bold;
+      font-size: 15px;
+      color: $base-black;
+    }
+
+    &__list {
+      display: none;
```
Comment (maksym-syniuk) [inline src/app/header/header.component.scss]: This logic with dropdown selection should be implemented with 
```ts 
<select formControlName="...">
        <option *ngFor="let category of categories" [value]="category">
          {{ category }}
        </option>
</select>
```
That would be in the future, when you will be implementing `Angular Reactive Forms`, so for now you can skip this. 
**Note**: Always remember, that when you want to hide important information and you use `display: none` or `visibility: hidden` or  other styles, every user can easily open devTools and display it.

**Review by iyarymovych-reenbit — APPROVED** [review]

File: src/app/breadcrumbs/breadcrumbs.component.ts (hunk 2 above)
Comment (Yuliia143) [reply src/app/breadcrumbs/breadcrumbs.component.ts:26]: Ok, done

File: src/app/breadcrumbs/breadcrumbs.component.ts (hunk 1 above)
Comment (Yuliia143) [reply src/app/breadcrumbs/breadcrumbs.component.ts]: Done

**Review by v-skochko — APPROVED** [review]
Approved!

**Review by maksym-syniuk — APPROVED** [review]

**Pull request merged**